baseplate/context/cqlmapper.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from cqlmapper.connection import Connection as CQLConnection
+
+from .cassandra import CassandraContextFactory
+
+
+class CQLMapperContextFactory(CassandraContextFactory):
```

Comment:
would you be OK just putting this in the `cassandra` module so it's all in one place? that might mean putting the import in the method call so it's not required for non-cqlmapper clients, but that's OK.